### PR TITLE
Fixed #53

### LINKED ISSUE #53: Weird UI Issue along with Console Warnings.
Hello, 

I tried installing passport and following the tutorial and the first time it all worked fine but later on, I broke something and decided to start fresh. I deleted the database and got a fresh copy of Laravel. 

Now when I tried to follow the exact same tutorial on the Docs and the video Taylor made i get some issues. I don't know what's causing them and I don't know how to fix them.

Here are the issues.

screen shot 2016-08-26 at 00 01 58

When i click the delete buttons i get this message

screen shot 2016-08-26 at 00 16 07

I have used a new database and i tried in incognito and cleared all my browsers cookies and cache. 

I have asked on IRC but no one can help me.

### PR DESCRIPTION
**Pull request by fwartner**
bildschirmfoto 2016-09-12 um 17 10 00

### CODE REVIEW

File: resources/assets/js/components/AuthorizedClients.vue [hunk 1]
```diff
@@ -76,8 +76,8 @@
              */
             getTokens() {
                 this.$http.get('/oauth/tokens')
-                        .then(response => {
-                            this.tokens = response.data;
+                        .then((response) => {
+                            this.tokens = response.json();
```
Comment (jbrooksuk) [inline resources/assets/js/components/AuthorizedClients.vue]: You don't need to do this if there is only one parameter.

Comment (fwartner) [inline resources/assets/js/components/AuthorizedClients.vue]: I don´t need to do what exactly?

**Comment by taylorotwell** [comment]
@themsaid isn't using `.data` still fine? @fwartner can you describe the actual issue?

**Comment by fwartner** [comment]
The issue (#53) was that when passport runs behind https, the ui gave us some weird stuff. Also the console.

I had the problem once in 5.3 with the ```response.data```.. After i changed it to ```response.json()```it worked.

**Comment by fwartner** [comment]
I think it has something to do with the vue-resource package..

**Comment by taylorotwell** [comment]
Yeah it's a vue-resource bug that they need to fix on their end. As far as I know there is a PR open on their side.

File: resources/assets/js/components/AuthorizedClients.vue (hunk 1 above)
Comment (jbrooksuk) [inline resources/assets/js/components/AuthorizedClients.vue]: To wrap the callback in brackets.

**Comment by fwartner** [comment]
@taylorotwell can´t find the pr you mentioned. Would you like to reference it?

**Comment by fwartner** [comment]
[Response Types](https://github.com/vuejs/vue-resource/blob/master/docs/http.md#response) HA! i was right! 😆

**Comment by themsaid** [comment]
Yeah I totally believe it's an issue in vue-resource, will need to dig deeper into that though.

**Comment by fwartner** [comment]
@themsaid check out the response-types of vue-resource. There´s no ```data``` object referenced..

**Comment by taylorotwell** [comment]
https://github.com/vuejs/vue-resource/issues/317

**Comment by themsaid** [comment]
@fwartner there's a getter for data that returns `this.body`

**Pull request closed without merging**